### PR TITLE
chore: PyTorch Samples - README.md update and notebook cleanup

### PR DESCRIPTION
**Pull request by shrinath-suresh**
**Description of your changes:**

This PR contains  the following changes

1. Added `prerequiste.md ` - which covers the prerequisite steps to be run on kubeflow before deploying the examples
2. Updated `README.md` with the instructions for gpu
3. Cifar10 and Bert notebook cleanup

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by jagadeeshi2i** [comment]
@googlebot I consent.

**Comment by Arvind-Ideas2IT** [comment]
@googlebot I consent.

File: samples/contrib/pytorch-samples/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+### Modify KFServing torchserve image
+
+Edit inferenceservice-config configmap
+
+```kubectl edit cm inferenceservice-config -n kubeflow```
+
+Update the following keys under `predictors->pytorch->v2` block
+
+```
+"image": "public.ecr.aws/y1x1p2u5/torchserve-kfs",
```
Comment (chauhang) [inline samples/contrib/pytorch-samples/prerequisites.md:14]: Is this different from 'pytorch/torchserve-kfs:0.4.0'

File: samples/contrib/pytorch-samples/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+### Creating custom notebook server
+
+In the kubeflow dashboard, create a new jupyter notebook server using following custom image
+
+    a. Open kubeflow dashboard
+    b. Select Notebooks and Click on New servers 
+    c. Add new notebook server by choosing custom image
+    d. Use the below custom image
+
+```public.ecr.aws/y1x1p2u5/jupyter:latest-cpu```
```
Comment (chauhang) [inline samples/contrib/pytorch-samples/prerequisites.md:86]: Let's include dockerfile for the customer jupyter notebook as well in a folder

**Review by chauhang — CHANGES_REQUESTED** [review]
Please see comments inline and also add links in readme for: 

1. PyTorch Vertex-AI Samples
2. In the main pytorch-kfp-components add link pointing to samples readme
3. In the cluster readme, include step for port forwarding as well

**Comment by ankan94** [comment]
@googlebot I consent.

File: samples/contrib/pytorch-samples/prerequisites.md (hunk 1 above)
Comment (jagadeeshi2i) [reply samples/contrib/pytorch-samples/prerequisites.md:14]: We will raise separate PR for requirement.txt. Once that is done we will change this to pytorch/torchserve-kfs:0.4.0

File: samples/contrib/pytorch-samples/prerequisites.md (hunk 2 above)
Comment (shrinath-suresh) [reply samples/contrib/pytorch-samples/prerequisites.md:92]: Done Geeta

**Review by chauhang — APPROVED** [review]

**Pull request merged**